evaluator/evaluate.go

```diff
@@ -233,7 +234,21 @@ func getPackages(tx *gorm.DB, data vmaas.UpdatesV2Response) (
 	return pkgByName, updatesByName, nil
 }
 
-func updateSystemPlatform(tx *gorm.DB, system *models.SystemPlatform, old, new SystemAdvisoryMap) error {
+func getPackageCounts(data vmaas.UpdatesV2Response) (int, int) {
+	installed := 0
```

Comment:
So can we directly use installed := len(data.UpdateList) here and save periodic incrementing?